.github/workflows/tests-clustermesh-upgrade.yaml

```diff
@@ -463,7 +463,10 @@ jobs:
           # interruption in such flows.
           cilium --context ${{ env.contextName1 }} connectivity test \
             --multi-cluster=${{ env.contextName2 }} --hubble=false \
-            --include-conn-disrupt-test --include-conn-disrupt-test-ns-traffic \
+            --include-unsafe-tests \
+            --include-conn-disrupt-test \
+            --include-conn-disrupt-test-ns-traffic \
```

Comment:
For some reason this test does not seem to be run: https://github.com/cilium/cilium/actions/runs/13890711080/job/38862133096#step:27:255

Reply:
Ah fair enough, I got confused with the conformance clustermesh tests that instead set `--nodes-without-cilium`. I think it is worthy enabling the N/S disruption test in this workflow as well, but we can leave that for a follow-up PR, given that it is unrelated from this one (and so that it is easy to troubleshoot possible problems).